### DISCUSSION TITLE
Learning R for complete beginners

### DISCUSSION THREAD

**Discussion by serpro69** [BryanJenksCommunity/FAQ#111] [Q&A] [answered]
Hi Bryan,

Any tips/suggestions you can give on how to approach learning R for a complete beginner? (Beginner both in R and programming in general.)
Any courses/resources/books/etc. that you can recommend to use?
Do you know any good decks for anki to learn R? (So far I've been only able to find two R-related decks on the anki website.)

Thanks :)

**Comment by tallguyjenks** [comment, marked as answer]
For anki i and many others think that making your own decks is the best approach so you retain only what you were struggling to retain

for starting resources start with the R for data science book, it's free and hosted online on github along with several other R books, there's sooo many free R books out there but R4DS is a great start

**Comment by serpro69** [comment]
Thanks :) I agree that making anki cards yourself is the best approach, but it does take a considerable amount of time, so that is not always possible unfortunately. 
Do you know any noteworthy courses on datacamp or other similar platforms?

For some reason I'm not able to mark the answer :/

**Reply by tallguyjenks** [reply]
true, but that focused effort on only the things you need help remembering is more bang for the buck and you waste less time studying unimportant details, its custom geared to how you best process information. highly recommend the "buy one cry once" mentality with effort investment into your own decks

i avoid datacamp because of their sexual harassment scandals towards female engineers and their CEO.
